mne/export/tests/test_export.py

```diff
@@ -396,6 +397,10 @@ def test_export_evokeds_to_mff(tmp_path, fmt, do_history):
     else:
         export_evokeds(export_fname, evoked, overwrite=True)
 
+    # test export from evoked directly
+    # XXX: this fails because "EvokedArray" has no export method?
+    # evoked[0].export(export_fname)
```

Comment:
@larsoner why does this not work for EvokedArray?

I added the `export` method to Evoked (see above) and wanted to add a quick test here ... but it fails :shrug: do I have to add the export method to some "super class"?

Reply:
unlike for raw and epochs, there is no "BaseEvoked"

Reply:
Evoked is effectively BaseEvoked, and EvokedArray inherits from it

https://github.com/mne-tools/mne-python/blob/a877fada44f4241e740dad3dee8ffeb1a50422e3/mne/evoked.py#L864

Knowing this, do you want to look further, or do you want me to look at the failure?

Reply:
Thanks! No need to look at the failure, this helps me.